readme.md

```diff
@@ -142,6 +142,20 @@ Please see the list below:
 
 Note: We have changed the return of `ecrecover` to be `uint160` because we use the `address` type for StarkNet addresses.
 
+## Docker :whale:
+
+Build the image from source:
+
+```bash
+docker build -t warp .
+```
+
+Run the container in the directory containing the target `.sol` files:
```

Comment:
Is it necessary to run warp in the same directory as the solidity file?

Reply:
No, it's not necessary. I rephrased the relevant part in the readme.md.
I added additional instructions on using `alias` such that the container feels exactly like the Warp binary.